lib/src/signed_video_h26x_common.c

```diff
@@ -1312,3 +1313,15 @@ signed_video_compare_versions(const char *version1, const char *version2)
 error:
   return status;
 }
+
+bool
+signed_video_is_start_of_stream(uint8_t *sei, size_t sei_size, SignedVideoCodec codec)
+{
+  h26x_nalu_t nalu = parse_nalu_info(sei, sei_size, codec, false, true);
+  if (nalu.nalu_type != NALU_TYPE_SEI) return false;
```

Comment:
Isn't it enough to simplify this function to
```
h26x_nalu_t nalu = parse_nalu_info(sei, sei_size, codec, false, true);
return nalu.start_of_sei;
```
Plus adding sanity checks on inputs.